src/ring_theory/int/basic.lean

```diff
@@ -86,8 +86,33 @@ end⟩
 instance : unique_factorization_monoid ℕ :=
 ⟨λ _, nat.irreducible_iff_prime⟩
 
+instance unique_units : unique (units ℕ) :=
```

Comment:
Would it make sense to move `unique_units` and `unique_add_units` to `data.nat.basic`, after the theorems `units_eq_one` and `add_units_eq_zero`?

Reply:
It would also make sense. That's where I had them first. Some maintainer should comment where it is preferred. It would compile as well.

Reply:
Sorry for being slow to react. I think it would be good to move that `nat` lemmas to `data.nat.basic`, and the `int` lemmas to `data.int.basic` (if that compiles?)